clang/test/Driver/clang-offload-bundler-zlib.c

```diff
@@ -1,4 +1,4 @@
-// REQUIRES: zlib
+// REQUIRES: zlib && !zstd
```

Comment:
Since zstd configurations become more prevalence, `zlib && !zstd` would essentially disable the test for increasingly more bots. But I guess this cannot be improved.